.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "arrow-parens": "off",
+      "no-use-before-define": "off",
+      "no-unused-vars": "off"
```

Comment:
- [ ] You seem to have added more rules here. Please use the exact rules stated [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) as they are shared by other students and keeps the project professional.